src/state/learnocaml_api.ml

```diff
@@ -166,6 +169,8 @@ module Conversions (Json: JSON_CODEC) = struct
         get ~token ["save.json"]
     | Update_save (token, save) ->
         post ~token ["sync"] (Json.encode Save.enc save)
+    | Git _ ->
+        assert false (* Reserved for the [git] client *)
```

Comment:
Not sure to understand the comment here.

Reply:
This is the function that constructs requests, and Learn-ocaml itself won't ever make this request, only receive it from the `git` command-line client.

Reply:
Oh I see! Thanks!